stan/math/prim/prob/bernoulli_lpmf.hpp

```diff
@@ -68,12 +69,12 @@ return_type_t<T_prob> bernoulli_lpmf(const T_n& n, const T_prob& theta) {
     if (sum == N) {
       logp += N * log(theta_dbl);
       if (!is_constant_all<T_prob>::value) {
-        ops_partials.edge1_.partials_[0] += N / theta_dbl;
+        ops_partials.edge1_.partials_[0] += N * inv(theta_dbl);
```

Comment:
What's up with this change?

Reply:
Oh this happened in a few places, why switch from `/` to `* inv()`?

Reply:
https://github.com/stan-dev/math/issues/2307

Reply:
I'm gonna add the static cast to double for the division here inf fvar's divide